docs/docs/codes/modifiers.mdx

```diff
@@ -30,13 +30,17 @@ These act like any other key code.
 
 ### Modifier Functions
 
+Modifier functions add one or more modifiers to a code.
```

Comment:
I just realised this sentence duplicates the first bullet point, but up to you whether you want to fix that.

Reply:
I think its fine to repeat the description like this. I see the list as describing each of the features of this functionality in more detail with examples.